release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -865,6 +865,13 @@ Support for the Broadcom BCM57504 network interface controller is now available
 
 With this release, the {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports both layer 2 switched and localnet switched topology networks.For more information about OVN-Kubernetes as a secondary network, see xref:../networking/multiple_networks/configuring-additional-network.adoc#configuration-ovnk-additional-networks_configuring-additional-network[Configuration for an OVN-Kubernetes additional network].
 
+[id="ocp-4-14-global-ip-forwarding-network-disabled"]
+==== Global IP forwarding is disabled on OVN-Kubernetes based cluster deployments
```

Comment:
Clusters upgrading to 4.14 won't be impacted by this change, i.e. IP forwarding behavior will remain unchanged (globally enabled).